packages/near-fast-auth-signer/src/components/SignMultichain/SignMultichain.tsx

```diff
@@ -147,10 +137,9 @@ function SignMultichain() {
             tokenAmount,
             feeProperties
           });
-
           setValid(true);
-          if (deserialize?.domain === event?.origin) {
-            await signMultichainTransaction(deserialize, transaction, feeProperties);
+          if (transaction?.domain === window.parent.origin) {
```

Comment:
is this part of code work? I saw an error message when I attempt to access `window.parent.origin`.